tests/integration/kubernetes/gha-run.sh

```diff
@@ -439,6 +439,35 @@ function cleanup_snapshotter() {
 	echo "::endgroup::"
 }
 
+function reset_k8s_images(){
```

Comment:
minor nit: Can we add a space between the `)` and `{` please to match the formatting of the rest of the file.